lib/IO/String.pm

```diff
@@ -36,10 +36,48 @@ TODO
 
 #| An L<IO::Handle> implementation that writes to memory.
 class IO::String:ver<0.1.0>:auth<hoelzro> is IO::Handle {
-    has @.contents;
+    has Str $.buffer = '';
+    has Int $.pos = 0;
+
+    multi method open(IO::String:D:) {
+        my Str $new-buffer = '';
+        $!buffer := $new-buffer;
+        $!pos = 0;
+    }
+
+    multi method open(IO::String:D: Str $buffer is rw) {
+        $!buffer := $buffer;
```

Comment:
I'm totally good with the feature if we have an explicit parameter like `:$bind`!